### PR TITLE
conf file

### PR DESCRIPTION
**Pull request by ahmedthinkdifferent**
check this conflict before merge

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,2 +1,5 @@
 FROM nginx:alpine
-COPY . /usr/share/nginx/html
\ No newline at end of file
+COPY . /usr/share/nginx/html
+WORKDIR /app/data
+RUN echo "this is test data" >> /app/data/app.txt
+VOLUME [ "/app/data" ]
```
Comment (unitcodelab) [inline Dockerfile:5]: what is need of volume

File: Dockerfile [hunk 2]
```diff
@@ -1,2 +1,5 @@
 FROM nginx:alpine
-COPY . /usr/share/nginx/html
```
Comment (ahmedthinkdifferent) [inline Dockerfile:2]: is there need to edit this line

Comment (unitcodelab) [reply Dockerfile:2]: yes

File: Dockerfile [hunk 3]
```diff
@@ -1,2 +1,5 @@
...
\ No newline at end of file
+COPY . /usr/share/nginx/html
+WORKDIR /app/data
```
Comment (unitcodelab) [inline Dockerfile:3]: why work dir?

**Review by unitcodelab — APPROVED** [review]
it's all good

File: Dockerfile (hunk 3 above)
Comment (ahmedthinkdifferent) [reply Dockerfile:3]: very important

**Pull request merged**